src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,17 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUMBER_OF_BALLS = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery[] lottery = new Lottery[NUMBER_OF_BALLS];
+        for (int i = 0; i < lottery.length; i++) {
+            lottery[i] = new Lottery().getRandomBall();
```

Comment:
Right now you create 3 lottery instances, you only need one - move lottery creation out of the scope of the loop